assets/schema_input.json

```
@@ -76,6 +76,14 @@
                 "meta": ["min_counts_gene"]
             }
         },
+        "oneOf": [
```

```suggestion
        "anyOf": [
```

Based on the documentation [here](https://json-schema.org/understanding-json-schema/reference/combining), `oneOf` would be an XOR, which is not what we want here